src/app.js

```diff
@@ -1 +1,30 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const { question } = require('./question');
+
+const main = async() => {
+  let filename = '';
+  let content = '';
+
+  while (!filename.length) {
+    filename = await question('Filename: ');
+  }
+
+  while (!content.length) {
+    content = await question('Content: ');
+  }
```

Comment:
Did you try to set the values immediately? => `const filename = await question('Filename: ');`

Reply:
Then it wouldn't ask again if input length is zero no?

Reply:
Right I didn't think of that. You can check the input it in the other module and utilize reject() with a message or something but this is a much less complicated solution 👍